sdk/tests/conformance2/glsl3/00_test_list.txt

```diff
@@ -28,6 +28,7 @@ shader-with-invalid-characters.html
 shader-with-mis-matching-uniform-block.html
 short-circuiting-in-loop-condition.html
 texture-offset-out-of-range.html
+texture-offset-uniform-texture-coordination.html
```

Comment:
Throughout this patch, please rename "coordination" to "coordinate".

I think it'd also be OK to move this new test to 2.0.1 if you think fixing the driver bug is not feasible in the short term.